bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/handler/DeconzBridgeHandler.java

```diff
@@ -175,7 +175,8 @@ private void parseAPIKeyResponse(AsyncHttpClient.Result r) {
         String url = buildUrl(config.getHostWithoutPort(), config.httpPort, config.apikey);
         return http.get(url, config.timeout).thenApply(r -> {
             if (r.getResponseCode() == 403) {
-                return Optional.ofNullable((BridgeFullState) null);
+                Optional<BridgeFullState> emptyBridgeFullState = Optional.empty();
```

Comment:
Would it not work to return it directly?

Reply:
I had some problems with type inference, so settled with this approach. I'm very interested in an elegant way of solving it. :-)

Reply:
What about:

```
return Optional.<BridgeFullState>empty();
```

Reply:
That's it, thanks! Beautiful.